.github/ISSUE_TEMPLATE/feature-openapi-react-query.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Feature request: openapi-react-query"
```

Comment:
@kerwanp Any unique descriptions / questions you’d like to add for openapi-react-query? (Same for feature request)?